namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-dev/resources/elasticache.tf

```diff
@@ -1,8 +1,4 @@
-################################################################################
-# HMPPS Pin Phone Monitor Application Elasticache for ReDiS
-################################################################################
-
-module "hmpps_pin_phone_monitor_elasticache_redis" {
+module "pcms_elasticache_redis" {
```

Comment:
Redis is a temporary data store, so don't care if this gets torn down and rebuilt